### PR TITLE
Fix for flakey tests for thread subcriptions.

### PR DESCRIPTION
**Pull request by burtonator**
## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/8225

## Test Plan

- This is a test so we just let it run.

## Deployment Plan

1. 

## Other Considerations

-

### CODE REVIEW

File: libs/model/test/subscription/thread-subscription-lifecycle.spec.ts [hunk 1]
```diff
@@ -89,10 +89,21 @@ describe('Thread subscription lifecycle', () => {
...
     const res = await query(GetThreadSubscriptions(), {
       actor,
       payload: {},
     });
+
+    expect(res).to.exist;
+    expect(res![0]).to.exist;
+    expect(res![1]).to.exist;
+
+    res![0]!.Thread = null!;
```
Comment (timolegros) [inline libs/model/test/subscription/thread-subscription-lifecycle.spec.ts:104]: Why are we modifying the response from the API and then testing it? Shouldn't we add the thread to the deep members check below instead?

Comment (burtonator) [reply libs/model/test/subscription/thread-subscription-lifecycle.spec.ts:104]: It's a good question. The resulting types are different.

I'm going to rework this and then we can test them properly:

https://github.com/hicommonwealth/commonwealth/issues/8243

The issue is that the types are different from the real thread vs what we're getting back from the API so the equals will not work.  

I'm not even sure how this test worked to begin with.

**Review by mzparacha — APPROVED** [review]
Approving in advance as hotfix pre #8243

File: libs/model/test/subscription/thread-subscription-lifecycle.spec.ts (hunk 1 above)
Comment (burtonator) [reply libs/model/test/subscription/thread-subscription-lifecycle.spec.ts:104]: @timolegros I"m going to resolve this but fix it in another PR.

**Review by Israellund — APPROVED** [review]

**Pull request merged**